### PR TITLE
WL-5103 Validate survey's email template before saving.

### PR DESCRIPTION
**Pull request by RebeccaMiller-Which**
capture

### CODE REVIEW

File: evaluations/impl/src/java/org/sakaiproject/evaluation/logic/EvalEvaluationSetupServiceImpl.java [hunk 1]
```diff
@@ -54,10 +52,16 @@
 import org.sakaiproject.evaluation.model.EvalTemplate;
 import org.sakaiproject.evaluation.utils.ArrayUtils;
 import org.sakaiproject.evaluation.utils.EvalUtils;
+import org.sakaiproject.evaluation.utils.TextTemplateLogicUtils;
 import org.sakaiproject.genericdao.api.search.Order;
 import org.sakaiproject.genericdao.api.search.Restriction;
 import org.sakaiproject.genericdao.api.search.Search;
 
+import freemarker.template.Configuration;
```
Comment (buckett) [inline evaluations/impl/src/java/org/sakaiproject/evaluation/logic/EvalEvaluationSetupServiceImpl.java]: Imports look unused?

**Review by buckett — CHANGES_REQUESTED** [review]
Look good, just the imports I think.

**Comment by RebeccaMiller-Which** [comment]
Have removed the unnecessary imports.

**Comment by buckett** [comment]
@wl-jenkins test this please

**Review by buckett — APPROVED** [review]

**Comment by buckett** [comment]
Failing build is related to other issue.

**Pull request merged**